### DISCUSSION TITLE
Release of 2.10.0

### DISCUSSION THREAD

**Discussion by luna215** [epam/ketcher#2720] [General]
Hi, 

When is Ketcher 2.10.0 set to release?

Thank you!

**Comment by ShangChien** [comment]
It seems that the official [demo](https://lifescience.opensource.epam.com/KetcherDemo/index.html?api_path=/v2) is already version 2.10.0. 👀

**Reply by luna215** [reply]
It was just released yestarday: https://github.com/epam/ketcher/releases/tag/v2.10.0
